src/webots/nodes/utils/WbTriangleMesh.hpp

```diff
@@ -57,6 +57,7 @@ class WbTriangleMesh {
     return mVertices[coordinateIndexAt(triangle, vertex, component)];
   }
   double normalAt(int triangle, int vertex, int component) const { return mNormals[3 * indexAt(triangle, vertex) + component]; }
+  bool isNormalAtCreased(int triangle, int vertex) const { return mIsNormalCreased[indexAt(triangle, vertex)]; }
```

Comment:
```suggestion
  bool isNormalCreased(int triangle, int vertex) const { return mIsNormalCreased[indexAt(triangle, vertex)]; }
```

Reply:
All these functions have the 'At' (e.g. normalAt, textureCoordinateAt, nonRecursiveTextureCoordinateAt, indexAt, ...) so why not for this one?

Reply:
That sounds a little bit odd to me... If we want to keep this "At" for consistency with the other methods, then, we should probably use it as a suffix. That would make "isNormalCreasedAt" which looks more consistent with other methods, although I am not super fan of the "At" suffix which doesn't say much. Maybe we should get rid of it for all methods...

Reply:
I am also in favor of removing it for all the functions :wink: